cluster-operator/src/main/resources/cluster-roles/021-ClusterRole-strimzi-cluster-operator-role.yaml

```diff
@@ -15,3 +15,9 @@ rules:
   - delete
   - patch
   - update
+- apiGroups:
```

Comment:
I wonder whether we should start adding comments for these things to explain why we need this access. It would make life easier for people reviewing these things (meaning we should have to answer less questions about it), but it would be somewhat difficult to maintain. Wdyt?

Reply:
I added some comments to clarify this.

Reply:
FYI: I think this is a good idea. But it is currently not possible easily. The processing of the RBAC YAMLs from Helm Chart into the other directories strips the comments. I will open a separate issue for this.